emuu-client/src/components/UserProfile/Profile.js

```diff
@@ -1,9 +1,90 @@
-import React from "react";
 import "./Profile.scss";
 import Feeds from "./Feeds";
 import UserInfo from "./UserInfo";
+import React, { useState , useEffect } from "react";
+import "../../Firebase.js";
+import {storage, db} from "../../Firebase.js";
+import { ref,getStorage, uploadBytes, getDownloadURL } from "firebase/storage";
+import AddIcon from '@mui/icons-material/Add';
+import { collection, getDocs } from "firebase/firestore";
 
 function Profile() {
+
+
+  // Store uploaded file
+  //const [file, setFile] = useState("");
+
+
+  //File upload
+  //function handleChange(event) {
+  //  setFile(event.target.files[0]);
+  //}
```

Comment:
Remove commented code before merge to main. It's dead code, does not need to be here.